src/pal/tests/palsuite/exception_handling/pal_sxs/test1/dlltest2.cpp

```diff
@@ -30,21 +30,30 @@ int DllTest2()
 
     PAL_TRY(VOID*, unused, NULL)
     {
-        volatile int* p = (volatile int *)0x22; /* invalid pointer */
+        volatile int* p = (volatile int *)0x22; // Invalid pointer
 
-        bTry = TRUE;    /* indicate we hit the PAL_TRY block */
-        *p = 13;        /* causes an access violation exception */
+        bTry = TRUE;                            // Indicate we hit the PAL_TRY block
+        *p = 2;                                 // Causes an access violation exception
 
         Fail("ERROR: code was executed after the access violation.\n");
+        return FAIL;
     }
     PAL_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
     {
         if (!bTry)
         {
             Fail("ERROR: PAL_EXCEPT was hit without PAL_TRY being hit.\n");
+            return FAIL;
```

Comment:
This added return FAIL here and at other two places below should not be needed. The Fail calls `PAL_TerminateEx(FAIL)` so it never returns.